### ISSUE TITLE
Supporting for detection of `meta[name="viewport"][content="user-scalable=no"]`

### ISSUE THREAD

**Issue by kagankan** [markuplint/markuplint#716]
>Warning: Usage of user-scalable=no can cause accessibility issues to users with visual impairments such as low vision. WCAG requires a minimum of 2× scaling; however, the best practice is to enable a 5× zoom.
https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag

Add a nodeRule to the presets.a11y.json to warn this.

# Examples

## Correct

```html
<meta name="viewport" content="width=device-width, initial-scale=1">
```

## Incorrect

```html
<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" > 
```

# References

https://dequeuniversity.com/rules/axe/4.4/meta-viewport
